runbotics/runbotics-scheduler/src/queue/schedule-process/schedule-process.controller.ts

```diff
@@ -46,7 +46,7 @@ export class ScheduleProcessController {
             throw new BadRequestException('Wrong process id');
         }
 
-        const scheduleProcessWithUser = { ...scheduleProcess, user: request.user, process};
+        const scheduleProcessWithUser = { ...scheduleProcess, user: request.user, process };
```

Comment:
imo change name to the one not including WITH 
& why with 'user' if it also contains 'process'?